Title: k8s_daemon_pod_containers: Fixed TypeError while inventorizing kubernetes daemon pod containers
Knowledge: doc
Level: 1
Component: inv
Class: fix
Compatible: compat
Edition: cre
State: unknown
Version: 1.6.0p9
Date: 1581408684

The error message {{TypeError: inv_k8s_daemon_pod_containers() got multiple values for keyword argument 'inventory_tree'}} was shown and has been fixed.
